### DISCUSSION TITLE
特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by riverod9** [Yowkees/keyball#366] [Q&A] [answered]
keyball44組み立て後、写真の赤丸のキーが反応しません。
はんだが足りないのかと思い、キーソケットのはんだを増やしたのですが解決しませんでした。
キーソケットがおかしいのかと思い、新しいものと交換をし画像の状態になったのですがキーが反応しません。
テンプラの可能性があるダイオードを教えていただけないでしょうか？
その他原因になる箇所ありましたらご指摘いただけますと幸いです。
![IMG_20230928_232212](https://github.com/Yowkees/keyball/assets/3508879/2fc524dd-bcf8-43d3-a0ad-090b05884939)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

赤丸キーに該当するダイオードは下記画像の緑丸のものになります。これがテンプラしていないかご確認ください。
少々半田の量が多いようですので、減らしていただくと半田不良が見つけやすいと思いますのでご確認お願いいたします。

以上、よろしくお願いいたします。
![KB44_keyfix1](https://github.com/Yowkees/keyball/assets/84209772/fbb74eef-646a-4cc1-931e-3e0368ed547f)

**Reply by riverod9** [reply]
ご対応ありがとうございます。
指摘のダイオードを確認したところ、テンプラしていることが分かりました。
以前他の質問にコメントされているのを拝見したのですが、片方の半田を溶かしたところダイオードがポロッと落ちてしまい、正しくはんだ付けできていないことが確認できました。
おかげさまで無事キーも反応するようになり、全てのキーが動くことを確認できました。
ご指摘いただきまして大変勉強になりました。ありがとうございました。
